vpype/write.py

```diff
@@ -177,6 +177,40 @@ def write_hpgl(vector_data: VectorData, output, size: Tuple[float, float]) -> No
     :param output: file object to write to
     :param size: size of the page in pixel
     """
+    # for plotters A2 and above we need to offset the coords (LL = -309, -210)
+    offset = [-309, -210]
+
+    # scale offset
+    offset = [int(offset[0] / 0.025), int(offset[1] / 0.025)]
+
+    # plotter units in mm
+    scale = 0.025
```

Comment:
This number is probably wrong. You mentioned that units are 0.025mm. Everything in vpype is CSS pixel, which are 1/96th of an inch. So the factor should rather be 1/0.025 unit/mm * 25.4 mm/in * 1/96 in/px = 127/12 unit/px ~= 10.58333... unit/px

Also, scale are most often understood as multiplicative, so I'd stick to that (in the code that follows).

Reply:
Ok, have written a function to do the scaling and offset — which is cleaner — and used this conversion factor with it.